docs/LV_client/quick_start/LabVIEW_run_examples.md

```diff
@@ -1,128 +1,212 @@
-## Run the monoDrive Client- Examples
+# LabView Client Examples
 
-### Closed Loop Example
-
-1. Click on the arrow on the top-left corner to start the client.
+## monoDrive Client Examples
 
-2. Move the car using the Steering and Throttle controls.
+After installing the monoDrive Client from the VI package manager, several
+pre-built examples will be available through the "Help" -> "Find Examples" 
+dialog in LabView. The `ni_monoDrive.lvproj` contains references to all the VIs 
+described here.
 
-3. Change the direction of the car using the forward control.
+For all of these examples, you will need to have either the monoDrive Simulator
+running or the monoDrive Scene Editor open and in "Play" mode.
 
-4. Apply brakes using the Brake control.
-
-5. Change the Weather profile using the up and down control.
+### Closed Loop Example
 
-6. Select a different position to start if desired.
+The Closed Loop example (`mono_closed_loop_example.vi`) allows the user to 
+control the vehicle's steering, throttle, brakes, and simulation weather in 
+real time. 
 
 <div class="img_container">
-    <img class='wide_img' src="https://github.com/monoDriveIO/documentation/raw/master/docs/LV_client/quick_start_img/CL_example.png"/>
+    <img class='wide_img' src="../imgs/CL_example.png" />
 </div>
 
-<p>&nbsp;</p>
+Prior to running the example, the user can:
 
+* **Select the weather profile by changing selected weather profile using th 
+Weather Profile Control.** There are various permutations of weather available 
+in the monoDrive Simulator. 
 
+* **Select the map by using the Map Selection Drop-down.** When ran, the 
+monoDrive simulator will switch to this map prior to beginning the simulation.
 
-### Replay Example
-
-1. Click on the arrow on the top-left corner to start the client.
+* **Select a different position using the Start Position Control.** monoDrive 
+maps come with various different start positions around the map. The number in 
```

Comment:
is it redundant "various" and "different"?

Reply:
Removed.